app/helpers/interruption_screen_helper.rb

```diff
@@ -7,6 +7,8 @@ def display_informative_text(informative_text, lettings_log)
       value = if argument["label"]
                 pre_casing_value = lettings_log.form.get_question(argument["key"], lettings_log).answer_label(lettings_log)
                 pre_casing_value.downcase
+              elsif argument["currency"]
+                ["£", ActionController::Base.helpers.number_to_currency(lettings_log.public_send(argument["key"]), delimiter: ",", format: "%n")].join("")
```

Comment:
do you need to prefix with `ActionController::Base.helpers`, I would of thought default view helpers would be available in this context?

Reply:
Interesting, I'll check - just copied this from somewhere else we do a similar thing.

Reply:
Yep, works great, thanks